examples/usecases/transformers-next-item-prediction-single-column.ipynb

```diff
@@ -0,0 +1,2974 @@
+{
```

Comment:
<p>why do we need to write a custom config file? Is that bcs you are getting error and you are unblocking yourself?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/1045/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>